examples/sol-anchor-contract/Anchor.toml

```diff
@@ -1,10 +1,9 @@
-[programs.devnet]
-example_sol_anchor_contract = "BZh3CP454Ca1C9yBp2tpGAkXoKFti9x8ShJLSxNDpoxa"
-
 [provider]
-cluster = "devnet"
+cluster = "localnet"
 wallet = "~/.config/solana/id.json"
 
+[programs.localnet]
+sol_anchor_contract = "GFPM2LncpbWiLkePLs3QjcLVPw31B2h23FwFfhig79fh"
+
 [scripts]
-install = "npm install"
-test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 scripts/test.ts"
+test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
```

Comment:
In this section you can force the program to be deployed at one address without needing the private key by doing 
`[[test.genesis]]
address = "stkBL96RZkjY5ine4TvPihGqW8UHJfch2cokjAPzV8i"
program = "./deps/cardinal-staking/target/deploy/cardinal_stake_pool.so"`

Reply:
Thanks for your help, this issue has been fixed.